core/gesture.ts

```diff
@@ -495,11 +484,11 @@ export class Gesture {
    */
   handleMove(e: Event) {
     this.updateFromEvent_(e);
-    if (this.isDraggingWorkspace_) {
+    if (this.workspaceDragger_) {
```

Comment:
Ah I see, getting rid of the state-tracking booleans also gives you better type checks. Nice.